### PR TITLE
Update de.h

### PR DESCRIPTION
**Pull request by avoidr**
Translate GROUP_CREATE_WITH_AUDIO
Translate PROXY_EDIT_HINT_IP

---
This change is [Reviewable](https://reviewable.io/reviews/utox/utox/1239)

### CODE REVIEW

**Comment by avoidr** [comment]
I'm not done, yet. I'll add `PROXY_EDIT_HINT_IP`, too.

**Comment by avoidr** [comment]
Done.

**Comment by redmanmale** [comment]
:lgtm:

---

Reviewed 1 of 1 files at r2.
Review status: 1 file needs 1 more reviewer, 1 assignee LGTMs are missing, missing Admin LGTM at current revision. (waiting on @undefined, @undefined, @undefined, @undefined, @undefined, @undefined, @undefined, @undefined, @undefined, @undefined, @undefined, @undefined, @undefined, @undefined, @undefined, @undefined, @undefined, @undefined, @undefined, @undefined, @undefined, @undefined, @undefined, @undefined, @undefined, @undefined, @undefined, @undefined, @undefined, @undefined, @undefined, @undefined, @undefined, @undefined, @undefined, @undefined, @undefined, @undefined, @undefined, @undefined, @undefined, and @undefined)

---



*Comments from [Reviewable](https://reviewable.io/reviews/utox/utox/1239#-:-LFaHf2BEeWdLuY0WUch:bnfp4nl)*

File: langs/de.h [hunk 1]
```diff
@@ -148,6 +148,9 @@ msgstr("Chatverlauf als Text exportieren")
 msgid(GROUPCHAT_JOIN_AUDIO)
 msgstr("Audiochat beitreten")
 
+msgid(GROUP_CREATE_WITH_AUDIO)
+msgstr("Aktiviere Audio")
```
Comment (cebe) [inline langs/de.h]: wäre nicht "Audio aktivieren" besser und konsistenter mit anderen Texten?

File: langs/de.h [hunk 2]
```diff
@@ -497,6 +500,9 @@ msgstr("Bitte füge mich deiner Kontaktliste hinzu.")
 msgid(CONTACT_SEARCH_ADD_HINT)
 msgstr("Freunde suchen/hinzufügen")
 
+msgid(PROXY_EDIT_HINT_IP)
+msgstr("IP Adresse")
```
Comment (cebe) [inline langs/de.h]: -> "IP-Adresse", siehe https://de.wikipedia.org/wiki/IP-Adresse

File: langs/de.h (hunk 1 above)
Comment (avoidr) [reply langs/de.h]: Stimmt, ist mir nicht aufgefallen.

**Comment by avoidr** [comment]
Done.

**Comment by cebe** [comment]
wow, what's wrong with the reviewable message?

**Pull request merged**